packages/cactus-plugin-bungee-hermes/src/main/json/openapi.json

```diff
@@ -28,8 +28,8 @@
       "PrivacyPolicyOpts": {
         "description": "identifier of the policy used to process a view",
         "type": "string",
-        "enum": ["pruneState"],
-        "x-enum-varnames": ["PruneState"]
+        "enum": ["pruneState", "singleTransaction"],
```

Comment:
@eduv09 These changes should be made in the tpl file not this one. Otherwise they'll disappear when the `codegen` re-renders this file from the tpl